docs/authentication/social-connections/spotify.mdx

```diff
@@ -1,65 +1,72 @@
+  ### Enable Spotify as a social connection in Clerk
 
-  ### Spotify + your Clerk app
+  1. In your **Clerk Dashboard**, go to **User & Authentication > [SSO Connections](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections)**.
+  1. Select **Add connection** and choose **For all users**.
+  1. In the **Choose provider** dropdown, search for **Spotify**, then select **Spotify** from the list.
+  1. The configuration options will appear below. Copy the **Authorized redirect URI**. Keep the modal and page open throughout the guide.
 
-  You can retrieve the **Redirect URI** from the Clerk Dashboard. But first, you need to enable Spotify as a social connection for your Clerk application.
+  ### Create your application in Spotify
 
-  Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Spotify**.
+  1. On a separate page, go to the [**Spotify Developer Dashboard**](https://developer.spotify.com/) and sign in to create a new Spotify application.
+  1. Once signed in, select your profile button in the top-right navigation and choose **Dashboard** from the dropdown menu. Select [**Create app**](https://developer.spotify.com/dashboard/create) on the dashboard page.
+  1. After being redirected to the application creation form, complete the fields for **App name** and **App description**. For **Redirect URIs**, paste the URL you copied from the Spotify modal in your **Clerk Dashboard**.
+  1. In the next section, you will connect Spotify to Clerk using your **Client ID** and **Client Secret**. For now, select **Settings**. Keep this page open for the next step.
 
-  ![The 'Social connections' page in the Clerk Dashboard. A red arrow is pointing to the Spotify toggle.](/docs/images/authentication-providers/spotify/enable-spotify.webp)
+  ### Set the Client ID and Client Secret in your Clerk Dashboard
 
-  Toggle on **Use custom credentials** and copy **Authorized redirect URI**. Keep this modal and page open.
+  1. In the Spotify application's settings page, scroll to the bottom and select **Edit**.
+  1. Scroll back to the top of the page and copy the **Client ID**.
+  1. Navigate back to your **Clerk Dashboard**. In the **Add connection for all users** modal under the **Configure Spotify** section, paste the string in **Client ID**.
+  1. Navigate back to the **Spotify Developer Dashboard**. Select the **View client secret** link and copy the string.
+  1. Paste the string in **Client Secret** and select **Add connection**. Once the settings are saved, you should see a green **`Used for sign-in`** label next to Spotify in the **Provider** list.
```

Comment:
We can make this more concise with the following suggestion, and we can also do our best to remove anything that's not necessary, as the dashboard's change a lot and we want to limit how much we rely on the current UI in our docs. so for example, "you should see a green **`Used for sign-in`** label next to Spotify in the **Provider** list." isn't technically necessary for this step, and it's more tech debt because things like "green" "use for sign-in" label, etc etc, are all things that could change. so let's try to limit how much we rely on the UI in the docs, because we have so much to keep up with already - if that makes sense.

```suggestion
  1. In the Spotify application's settings page, save the **Client ID** and **Client secret** somewhere secure. To reveal the **Client secret**, you must select the **View client secret** button.
  1. Navigate back to your **Clerk Dashboard** where the modal should still be open, and paste these values into the respective fields.
  1. Select **Add connection**.
  
  > [!NOTE]
  > If the modal or page is not still open, navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Spotify** connection. Under **Use custom credentials**, you can paste the **Client ID** and **Client Secret** into their respective fields.
```